lib/LaTeXML/Post.pm

```diff
@@ -447,7 +447,7 @@ sub convertXMTextContent {
   foreach my $node (@nodes) {
     if ($node->nodeType == XML_TEXT_NODE) {
       my $string = $node->textContent;
-      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
```

Comment:
On 08/11/2015 08:30 AM, Deyan Ginev wrote:
> In lib/LaTeXML/Post.pm
> <https://github.com/KWARC/LaTeXML/pull/8#discussion_r36738894>:
>
>> @@ -447,7 +447,7 @@ sub convertXMTextContent {
>>    foreach my $node (@nodes) {
>>      if ($node->nodeType == XML_TEXT_NODE) {
>>        my $string = $node->textContent;
>> -      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
>
> Wait, I take this back. @brucemiller <https://github.com/brucemiller>
> you confused me :> So Michael went ahead and merged the PR we are
> commenting on top of some time ago, a bit prematurely. That's where the
> snippet in question "lived".
>
> However, the last activity on the KWARC/LaTeXML repo is me merging over
> brucemiller/LaTeXML, which undid the prior change and synced with your
> work. Here is the exact current state:
>
> https://github.com/KWARC/LaTeXML/blob/master/lib/LaTeXML/Post.pm#L452

Yep; that looks right. MathML needs those nbsp's!

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KWARC/LaTeXML/pull/8/files#r36738894>.
>